crates/astria-bridge-withdrawer/local.env.example

```diff
@@ -35,7 +35,11 @@ ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_KEY_PATH=/path/to/priv_sequencer_key.json
 ASTRIA_BRIDGE_WITHDRAWER_FEE_ASSET_DENOMINATION="nria"
 
 # The asset denomination being withdrawn from the rollup.
-ASTRIA_BRIDGE_WITHDRAWER_ROLLUP_ASSET_DENOMINATION="nria",
+ASTRIA_BRIDGE_WITHDRAWER_ROLLUP_ASSET_DENOMINATION="nria"
+
+# The hex-encoded bridge address corresponding to the bridged rollup asset on the sequencer.
+# Should match the bridge address in the geth rollup's bridge configuration for that asset.
+ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_BRIDGE_ADDRESS=""
```

Comment:
i want to add these as immutable vars to the bridge contract instead of config values, will do in a follow up.